siliconcompiler/tools/yosys/yosys.py

```diff
@@ -368,6 +368,11 @@ def get_dff_liberty_file(chip):
     if dff_liberty:
         return dff_liberty[0]
 
+    mainlib = chip.get('asic', 'logiclib')[0]
```

Comment:
At some point in the future, we may want to add a Chip helper method for fetching this 'mainlib' value, and clarify if 'first entry' is a good criteria.

Reply:
Helper functions would be great, but those can wait.